src/components/SecuritySection.vue

```diff
@@ -0,0 +1,203 @@
+<!--
+  - SPDX-FileCopyrightText: 2022 Carl Schwan <carl@carlschwan.eu>
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+  -->
+
+<template>
+	<SettingsSection :title="t('end_to_end_encryption', 'End to End Encryption')"
+		:description="encryptionState">
+		<Button :disabled="!hasKey" type="warning" @click="startResetProcess()">
+			{{ t('end_to_end_encryption', 'Reset End to End Encryption') }}
+		</Button>
+
+		<div v-if="shouldDisplayWarning && hasKey" class="notecard warning" role="alert">
+			<p>{{ t('end_to_end_encryption', 'Please read carefully before resetting your end-to-end encryption keys') }}</p>
+			<ul>
+				<li>{{ t('end_to_end_encryption', 'Once you end to end encryption reset, all files stored in your encrypted folder will be inaccessible') }}</li>
+				<li>{{ t('end_to_end_encryption', 'You should only reset your end to end encryption when you have lost your secure key words (mnemonic)') }}</li>
+				<li>{{ t('end_to_end_encryption', 'Check on all connected devices, if you can retrieve your mnemonic') }}</li>
+				<li>{{ t('end_to_end_encryption', 'After deleting the keys any still existing device might cause problems, therefore please re-setup the accounts.') }}</li>
+			</ul>
+
+			<CheckboxRadioSwitch :checked.sync="deleteEncryptedFiles" type="switch" class="margin-bottom">
+				{{ t('end_to_end_encryption', 'Delete existing encrypted files') }}
+			</CheckboxRadioSwitch>
+
+			<Button type="error" @click="showModal">
+				{{ t('end_to_end_encryption', "Confirm") }}
+			</Button>
+
+			<Modal v-if="modal"
+				size="small"
+				class="modal"
+				@close="closeModal">
+				<div class="modal-container">
+					<p>
+						{{ t('end_to_end_encryption', 'This is the final warning: Do you really want to reset your keys?') }}
+					</p>
```

Comment:
```suggestion
					<p><strong>
						{{ t('end_to_end_encryption', 'This is the final warning: Do you really want to reset your keys?') }}
					</strong></p>
```

Same as the first line of the block, bolding it for emphasis.